### PR TITLE
Adressing comments from Tom Moreau

### PR DESCRIPTION
**Pull request by AdrianPadilla**

### CODE REVIEW

File: model/src/main/java/oracle/kubernetes/weblogic/domain/v2/ServerSpecV2Impl.java [hunk 1]
```diff
@@ -167,7 +165,6 @@ public String toString() {
         .appendSuper(super.toString())
         .append("server", server)
         .append("clusterLimit", clusterLimit)
-        .append("effectiveRestartVersion", effectiveRestartVersion.toString())
         .toString();
```
Comment (moreaut) [inline model/src/main/java/oracle/kubernetes/weblogic/domain/v2/ServerSpecV2Impl.java]: append cluster too?

File: model/src/main/java/oracle/kubernetes/weblogic/domain/v2/ServerSpecV2Impl.java [hunk 2]
```diff
@@ -195,79 +191,6 @@ public int hashCode() {
         .appendSuper(super.hashCode())
         .append(server)
         .append(clusterLimit)
-        .append(effectiveRestartVersion)
         .toHashCode();
```
Comment (moreaut) [inline model/src/main/java/oracle/kubernetes/weblogic/domain/v2/ServerSpecV2Impl.java]: append cluster too?

File: model/src/main/java/oracle/kubernetes/weblogic/domain/v2/ServerSpecV2Impl.java (hunk 1 above)
Comment (AdrianPadilla) [reply model/src/main/java/oracle/kubernetes/weblogic/domain/v2/ServerSpecV2Impl.java]: yep, done

File: model/src/main/java/oracle/kubernetes/weblogic/domain/v2/ServerSpecV2Impl.java (hunk 2 above)
Comment (AdrianPadilla) [reply model/src/main/java/oracle/kubernetes/weblogic/domain/v2/ServerSpecV2Impl.java]: updated along with equals method too

**Pull request merged**